docs/guides/earning-cred.md

```diff
@@ -1,42 +1,41 @@
 
+### 🏄🏾 Cred Flowing
+Every contributor gets a Cred score. Your Cred Score is the amount of Cred that is flowing *through* your pond - the amount entering is the same amount that is simultaneously leaving, flowing to other community members. Each of your contributions is interconnected with other community contributions across different platforms. Contributions are even connected through time - today's contributions connect to contributions from yesterday, last week, etc.
 
+To put it differently, you are part of an environment that sustains itself across time where water is key. Water will flow in different ways. It will flow between contributor ponds (Cred Flowing), fall from the sky as rain (Cred Minting) and evaporate from everyone's ponds over time (Cred Radiation).
 
-### 🏄🏾Cred flowing
-All the contributions you and your community make are on the same Contribution Graph. Each of your contributions is interconnected with other community contributions, across different platforms. Contributions are even also connected through time - today's contributions connect to contributions from yesterday, last week, etc
-
-To put it differently, you are part of an environment that sustains itself across time for which water is key. In your pond, water will not only flow from the sky. Indeed, being connected with other ponds will make the exchange of water possible. It might be drops or a steady stream depending on the size of your connections. And at any point the sky&sun will make sure that some of your drops evaporate to keep everything in balance
-
-
-### 💡 At a conceptual level 
+### 💡 At a Conceptual Level 
 
 When you react to a message with <img width="20" alt="SourceCred" src='https://sourcecred.io/img/favicon.png' /> in one of the plugins :
 
   - it creates a new reaction node in the graph, which mints Cred 
   - it creates an edge from the person reacting (you), to the reaction node
   - it creates an edge pointing from the reaction node to the message receiving the reaction
   
-Minting new cred is also a change for the entire Contribution Graph. It will therefore induce a change on how Cred flows for the others so as to keep everything in balance. Again, we need to make sure that water is circulating well with no congestions. 
+Minting new cred is also a change for the entire Contribution Graph. It will therefore induce a change on how Cred flows to others so as to keep everything in balance. Again, we need to make sure that water is circulating well with no congestions. 
 
-You want more on this? [Here] is a broader explanations of the concepts behing SourceCred by Evan Miyazono from [Protocol Labs Research] 
+You want more on this? [Here] is a broader explanation of the concepts behind SourceCred by Evan Miyazono from [Protocol Labs Research] 
```

Comment:
This might work better at the beginning of the page? It works here too, but I'm wondering about the person who wants to skip to the more detailed explanation. We don't want them to have to read all the way through to get to this, which they might not do.

Reply:
I hear what you're saying, but someone looking for more detailed information knows that the 'more links here' section is typically at the bottom of a page and can easily scroll or skim. I don't want to disrupt the flow of the writing y adding a 'This is what we're talking about and here's more info if you want' section at the start. Instead, I'll make a Want More? header for this section so it's easily seen